ui/app/src/components/RejectDialog/RejectDialogContainer.tsx

```diff
@@ -180,7 +165,7 @@ export function RejectDialogContainer(props: RejectDialogContainerProps) {
       onUpdateChecked={updateChecked}
       onCloseButtonClick={onCloseButtonClick}
       onReject={onReject}
-      classes={useStyles()}
+      classes={classes}
```

Comment:
Classes not used here at all? move to the component that consumes and remove as prop